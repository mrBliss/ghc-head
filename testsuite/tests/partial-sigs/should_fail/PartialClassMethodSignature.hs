{-# LANGUAGE PartialTypeSignatures #-}
module PartialClassMethodSignature where


class Foo a where
  foo :: (Eq a, _) => a -> _

