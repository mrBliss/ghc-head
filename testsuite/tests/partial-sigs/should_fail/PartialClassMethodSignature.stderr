
PartialClassMethodSignature.hs:6:17:
    The type signature of a class method cannot be partial:
    foo :: (Eq a, _) => a -> _
    In the class declaration for ‘Foo’
